Common.php

```diff
@@ -0,0 +1,19 @@
+<?php
+function response($input)
+{
+    #error_reporting(0);
+    if (is_array($input[0])) {
+        foreach ($input as $array) {
```

Comment:
you don't need to iterate over array to produce each json line.
json_encode() will create string from array of arrays